tests/functional/aws-node-sdk/test/bucket/testBucketVersioning.js

```diff
@@ -169,3 +167,26 @@ describe('aws-node-sdk test bucket versioning', function testSuite() {
         });
     });
 });
+
+
+describe('bucket versioning for ingestion buckets', () => {
+    const Bucket = `ingestion-bucket-${Date.now()}`;
+    before(done => s3.createBucket({
+            Bucket,
+            CreateBucketConfiguration: {
+                LocationConstraint: 'us-east-2:ingest',
+            },
+        }, done));
+
+    after(done => s3.deleteBucket({ Bucket }, done));
+
+    it('should not allow suspending versioning for ingestion buckets', done => {
+        s3.putBucketVersioning({ Bucket, VersioningConfiguration: {
+            Status: 'Disabled'
+        } }, err => {
+            assert(err, 'Expected error but got success');
+            assert.strictEqual(err.InvalidBucketState);
```

Comment:
Can you get rid of the second argument implicitly to check if defined? (real question) otherwise it's missing :)

Reply:
Ah right.